config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openstack-k8s-operators/watcher-operator
+  newName: quay.io/dviroel/watcher-operator
```

Comment:
I guess this change in newName was unintended

Reply:
ouch
thanks